DataFormats/METReco/src/CSCHaloData.cc

```diff
@@ -54,7 +54,7 @@ int CSCHaloData::NumberOfHaloTracks(HaloData::Endcap z) const
   int n = 0 ;
   for(unsigned int i = 0 ; i < TheTrackRefs.size() ; i++ )
     {
-      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs, i ) ;
+      edm::Ref<reco::TrackCollection> iTrack( TheTrackRefs[i] ) ;
```

Comment:
For example, the RefVector might have 5 entries and the TrackCollection might have 1000 tracks. The 3rd element of the RefVector might reference the 500th track. The existing constructor is passing an index into the RefVector into a function that expects and index into the TrackCollection, In this example, passing 3 instead of 500.